### PR TITLE
Add MrBigFatDick. Fix SexLikeReal description for some scenes.

### PR DESCRIPTION
**Pull request by puc9**
Fix SexLikeReal description for some scenes.
Add URL to the output in case SexLikeReal redirects ours (for example slr/sceneId gets redirected to slr/title-sceneId)
Added YML schema comment at the top to enable VSCode (and some other editors) to validate while typing and offer suggestions.
See: [https://developers.redhat.com/blog/2020/11/25/how-to-configure-yaml-schema-to-make-editing-files-easier#schema_association_without_a_file_name_pattern](url)

### CODE REVIEW

File: scrapers/MrBigFatDick.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      - mrbigfatdick.com/videos/
+    scraper: sceneScraper2
+    queryURL:  https://backend.mrbigfatdick.com/api/public/updates/{url}
+    queryURLReplace:
+      url:
+        - regex: '.+/videos/([^/]+?)/?$'
+          with: $1
+
+debug:
+  printHTML: false
```
Comment (bnkai) [inline scrapers/MrBigFatDick.yml:16]: not needed can be removed

File: scrapers/MrBigFatDick.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        Name:
+          selector: tags.#.fullName
+      Performers:
+        Name:
+          selector: models.#.fullName
+          postProcess:
+            - map:
+                Mrbigfatdick: "Mr. Big Fat Dick"
+      Image:
+        selector: image3840
```
Comment (bnkai) [inline scrapers/MrBigFatDick.yml:37]: the 4k image is 3x the size of the 1080p and looks like an upscale to me ?
```yaml
      Image: image1920
```

File: scrapers/MrBigFatDick.yml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+      Details:
+        selector: description
+
```
Comment (bnkai) [inline scrapers/MrBigFatDick.yml:40]: the studio name is missing?
```yaml
      Studio:
        Name:
          fixed: Mr Big Fat Dick
```

File: scrapers/MrBigFatDick.yml [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  - action: scrapeJson
+    url:
+      - mrbigfatdick.com/videos/
+    scraper: sceneScraper2
+    queryURL:  https://backend.mrbigfatdick.com/api/public/updates/{url}
+    queryURLReplace:
+      url:
+        - regex: '.+/videos/([^/]+?)/?$'
+          with: $1
+
```
Comment (bnkai) [inline scrapers/MrBigFatDick.yml:14]: sceneByName support
```yaml
sceneByName:
  action: scrapeJson
  queryURL: https://backend.mrbigfatdick.com/api/search/{} # search doesnt like spaces, use _ instead when quering
  scraper: sceneSearch
sceneByQueryFragment:
  action: scrapeJson
  queryURL: "{url}"
  scraper: sceneScraper2

jsonScrapers:
  sceneSearch:
    scene:
      Title: updates.#.fullName
      URL:
        selector: updates.#.permaLink
        postProcess:
          - replace:
              - regex: ^
                with: "https://backend.mrbigfatdick.com/api/public/updates/"
      Image: updates.#.image620
      Date:       
        selector: updates.#.publishDate
        postProcess:
          - parseDate: 2006-01-02 15:04:05
```

File: scrapers/MrBigFatDick.yml (hunk 3 above)
Comment (bnkai) [inline scrapers/MrBigFatDick.yml:40]: Also `URL`, needed if we add the sceneByName 
```yaml
      URL:
        selector: permaLink
        postProcess:
            - replace:
              - regex: ^
                with: "https://www.mrbigfatdick.com/videos/"
```

**Comment by bnkai** [comment]
>Fix SexLikeReal description for some scenes

Can you provide some URLs with the issue? I can't find a test case to verify.

**Comment by puc9** [comment]
For SLR: Search for scene 3018. There are a few that have that extra text and link at the beginning of the description.

I'll fix the other one soon

File: scrapers/MrBigFatDick.yml (hunk 3 above)
Comment (puc9) [reply scrapers/MrBigFatDick.yml:72]: Used the same name as the one in the stashdb (no spaces)

**Pull request merged**